static/index.html

```diff
@@ -42,20 +42,19 @@
 
 <body>
   <div class="bg"></div>
-  <div class="container">
-    <div class="top">
+  <article class='container'>
+
+    <header>
       <div class="login">
-        <span class="logintitle">Login with</span>
-        <a href="/login/">
-          <img src="/static/images/GitHub-Mark-120px-plus.png" width="30" />
-        </a>
+        <p>Login with <a href="/login/">
+          <img src="/static/images/GitHub-Mark-120px-plus.png"
+               alt='GitHub logo linked to login'
+               width="30" /></a>
       </div>
-    </div>
-    <div class="heading">
       <img src="/static/images/logo_world_driven.svg" class="logo" alt="World driven logo">
-      
-    </div>
-    <center><h6>Trustable service landscape</h6></center>
+      <h4>Trustable service landscape</h4>
```

Comment:
I had this one very small because I'm not so sure about that tagline.

Somehow it describes the long (long long) term idea pretty good. Right now I'm more into getting feedback about the basic auto-merge idea.
So presenting mainly the auto merger and if this is of interest continue with Phase 2 and 3.
Does that make sense?

In that context, the tagline does not fit so well.
What do you think?

So the change is fine, just wanted to give some background.

Reply:
Yeah I don't love the tagline either. If we come up something better before this merges, which will be a while due to the pending merges prior to this that I'll need to incorporate thus resetting the timer, I'll update it; if not, I'll probably just remove it for now.

I agree with focusing on the auto-merge for now. I'd rather do one thing great that loads of things okay.

Reply:
I'm going to leave some slogan ideas below - I'll just add them here as I think of them rather than a new comment every time:

Collaboration Through Automation
Driving Development

Reply:
Hm, not yet convinced:

- Automation supported collaboration (in yours I see the automation part too strong. Not sure if it works in English, in German, this could be a pun)
- Driving Development would be a bit too much `Drive` in my opinion :-)

I thought about: When we go somehow in the 'tab' idea (for each phase a something like a tab), we could also adapt the tagline for each phase.
So Phase 1 more centered around the automated merging / collaboration
Phase 2 more a service idea tag line ...